script/build

```
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -e
+RESULT=0
+
+# Fail if we're not using php71+
+if ! (php --version | grep -q "PHP 7.") ; then
+    echo "You're not using PHP 7.1+, please install it."
+    php --version
+    exit 1
+fi
+
+# setup and run composer
+if [ -n "$OAUTH" ]; then
+    composer config -g github-oauth.github.com $OAUTH
+fi
+
+composer --no-ansi --no-interaction install
+
+# Make sure the reports directory exists
+if [ ! -d "build/reports" ]; then
+    mkdir -p build/reports
+fi
+
+# Show php version for debugging
+php --version
+
+# Run PHPUnit
+vendor/bin/phpunit || RESULT=1
+
+# Run PHPCS
+vendor/bin/phpcs --report=checkstyle --report-file=build/reports/checkstyle.xml || RESULT=1
```

```suggestion
vendor/bin/phpcs || RESULT=1
```
Reports files are not used